web/src/bootstrap_typeahead.ts

```diff
@@ -805,6 +807,19 @@ export class Typeahead<ItemType extends string | object> {
                     this.openInputFieldOnKeyUp();
                 }
                 if (e.key === "Backspace") {
+                    if (
+                        the(this.input_element.$element).id === "stream_message_recipient_topic" &&
+                        this.input_element.$element.val() === ""
```

Comment:
Seems odd to add this handler for a specific input here but the test was failing if we were adding a separate handler due to more than one handler the same event which makes sense given it is better to have a single evnet handler for a element to avoid handling order of execution.